mne/io/edf/edf.py

```diff
@@ -520,7 +524,7 @@ def _edf_str(x):
     return x.decode('latin-1').split('\x00')[0]
 
 
-def _read_edf_header(fname, exclude):
+def _read_edf_header(fname, exclude,misc=[]):
```

Comment:
Same here: don't use mutable objects as default arguments

Reply:
@hoechenberger as of now the issue stands just to support `µV ` as a valid unit for microvolt to resolve the scaling problem referred to in #9186 , and I plan to do away with the `misc` implementation entirely as it makes the loading process complicated (and also fails a lot of CI tests which pass data with other without `misc` . However, if `misc` is to be implented finally I will make the requested changes in code (implementation, style and documentation) . Currently waiting for @larsoner s opinion, to resolve #9186 if only adding support for an extra symbol would do the job, and the `misc` parts can be ignored entirely.

Reply:
@agramfort already gave his OK, so you could already implement/revert to the much simpler case. @larsoner can then directly merge if he's happy when he comes online.

Reply:
And don't forget to add your contribution to the changelog!